src/classes/PobControllerExt.cls

```diff
@@ -1,4 +1,9 @@
-public with sharing class PobControllerExt {
+public class PobControllerExt {
```

Comment:
Missing or incorrect sharing behavior on `PobControllerExt`. Please update the sharing clause to `with sharing`.